dev/io.openliberty.org.jboss.resteasy.common.ee11/src/io/openliberty/org/jboss/resteasy/common/client/LibertyResteasyClientBuilderImpl.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2022 IBM Corporation and others.
+ * Copyright (c) 2025 IBM Corporation and others.
```

Comment:
```suggestion
 * Copyright (c) 2022, 2025 IBM Corporation and others.
```
You can't get rid of the older copyright year.

Reply:
Done

Reply:
You replaced the Copyright with the JBoss one.  This is an IBM class as you can see with the io.openliberty package name prefix.  It should be put back to the IBM Copyright and have the dates be corrected as I originally had in the comment.  We borrow a lot of code from the JBoss class, so it might make sense to include that one after the IBM Copyright possibly.

Reply:
Done